GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -862,6 +929,17 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
                             replay.AddDecorationWithGrowing(new CircleDecoration( 200, (start, end), "rgba(49, 71, 0, 0.2)", new PositionConnector(effect.Position)), end);
                         }
                     }
+                    // Shockwaves
+                    if (log.CombatData.TryGetEffectEventsByGUID(EffectGUIDs.HarvestTempleMordremothShockwave1, out IReadOnlyList<EffectEvent> shockwaves))
```

Comment:
No Src check? There are a log of Void Amalgamate agents in the fight, every each one of them is going to add the shockwave circles with this code.

Reply:
No scratch that comment, it is just the github diff hiding under which case that code was.